jcl/source/common/JclSimpleXml.pas

```diff
@@ -629,7 +629,7 @@   TXMLVariant = class(TInvokeableVariantType)
       const Name: string; const Arguments: TVarDataArray): Boolean; override;
     function GetProperty(var Dest: TVarData; const V: TVarData;
       const Name: string): Boolean; override;
-    function SetProperty(const V: TVarData; const Name: string;
+    function SetProperty(var V: TVarData; const Name: string;
```

Comment:
Why change this to `var`? The `SetProperty` method does not change the value of `V` so there is no reason to make it a `var` parameter